### PR TITLE
Reformatted the scrapper code and now the data can be saved as a csv file

### PR DESCRIPTION
**Pull request by gp201**
TODO:
#1

### CODE REVIEW

**Comment by HimaSava** [comment]
One point that I want to be changed. 

Can you change the date format? Currently, it is (yyyy-mm) can we make it (mmm-yyyy) (eg.: Jan-2023).

The table format is actually exactly I had in mind 😄

**Comment by HimaSava** [comment]
Review point:

Test the code for multiple years. According to your code the excel will be re-written after processing data of each year.

**Comment by gp201** [comment]
> One point that I want to be changed.
> 
> Can you change the date format? Currently, it is (yyyy-mm) can we make it (mmm-yyyy) (eg.: Jan-2023).
> 
> The table format is actually exactly I had in mind :smile: 

I think it will be better to stick to the [ISO 8601 date format](https://en.wikipedia.org/wiki/ISO_8601#Calendar_dates).

File: scrapper.py [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+# set the year range for data
+START_YEAR = 2020
+END_YEAR = 2022
+
+# set the month range for data
+START_MONTH = 1
+END_MONTH = 12
+
+# set principal commodity code
+PRINCIPAL_COMMODITY_CODE = 1
```
Comment (gp201) [inline scrapper.py:19]: Move this to a file known as constant.py?

File: scrapper.py [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+URL = "https://tradestat.commerce.gov.in/meidb/brc.asp?ie=i"
+DATABASE_FILE = "data.csv"
+
+# set the year range for data
+START_YEAR = 2020
+END_YEAR = 2022
+
+# set the month range for data
+START_MONTH = 1
+END_MONTH = 12
```
Comment (gp201) [inline scrapper.py:16]: Instead of defining these as constants, we should add scripts to take it only additional data and not process all years

**Pull request closed without merging**